### PR TITLE
DOC : Update plugins page

### LINKED ISSUE #1881: Missing io.plugins function
On the [plugins page](https://github.com/scikit-image/scikit-image/blob/master/doc/source/user_guide/plugins.txt#L52) of the user guide, I found few uses of the function `skimage.io.plugins()` which is not available in the package now. I see that the `skimage.io.find_available_plugins()` does pretty much the same job. So to fix the guide, should we just replace `plugins()` with `find_available_plugins` in the guide?

I was not able to track the history of the file and see why was the function removed because the file was migrated from `io.plugins._plugins.py`.

### PR DESCRIPTION
**Pull request by OrkoHunter**
Fix #1881 

Updated old interpreted outputs.

### CODE REVIEW

**Comment by OrkoHunter** [comment]
Shouldn't the cutoff for coveralls be a little more? It failed for -0.0%

**Comment by soupault** [comment]
@OrkoHunter we'd like to ask you to prepare couple of tests for the user guide to increase the coverage. (just kidding)

File: doc/source/user_guide/plugins.txt [hunk 1]
```diff
@@ -49,17 +49,25 @@ Any plugin in the ``_plugins`` directory is automatically examined by
...
+   'gdal': ['imread', 'imread_collection'],
+   'gtk': ['imshow'],
+   'imageio': ['imread', 'imsave', 'imread_collection'],
+   'imread': ['imread', 'imsave', 'imread_collection'],
+   'matplotlib': ['imshow', 'imread', 'imread_collection'],
+   'pil': ['imread', 'imsave', 'imread_collection'],
+   'qt': ['imshow', 'imsave', 'imread', 'imread_collection'],
+   'simpleitk': ['imread', 'imsave', 'imread_collection'],
+   'test': ['imsave', 'imshow', 'imread', 'imread_collection'],
+   'tifffile': ['imread', 'imsave', 'imread_collection']}
```
Comment (soupault) [inline doc/source/user_guide/plugins.txt]: I don't have `imageio` in my installation. Anyways, the full output of the command is kind of redundant for a documentation, so I'd suggest to keep those 5 most common plugins only. What do you think?

Comment (OrkoHunter) [inline doc/source/user_guide/plugins.txt]: Sounds reasonable. Let's just keep the original ones.

**Comment by OrkoHunter** [comment]
@soupault And now it's hung !

**Comment by soupault** [comment]
@OrkoHunter yeah, `coveralls.io` seems to be under maintenance. The PR does nothing to affect coverage though.
Merging, thanks!

**Comment by OrkoHunter** [comment]
Cool, thanks!

**Pull request merged**